lib/dialects/pgnative/index.js

```diff
@@ -0,0 +1,58 @@
+// PostgreSQL Native Driver (pg-native)
+// -------
+const Client_PG = require('../postgres');
```

Comment:
Why use `.native` on `pg` and not `pg-native` as per https://github.com/brianc/node-pg-native? Was separate driver deprecated?

Reply:
Brian has it wrapped up inside of `pg` directly (dependency) so it seemed like the better approach since he *tries* to make them as compatible as possible.

Reply:
Should we drop pg-native dependency then, which seems to have been added?

Reply:
It only needs to be present as a `peerDependency`. I'm not sure whether that should be at the library level or at the application code level. What're your thoughts @kibertoad ?

Reply:
Makes sense, should also be optional peerDependency for knex as is the case for all other drivers.

Reply:
Excellent. That's my current setup AFAIK. It is included as a `devDependency` like the other dialects. Is that okay as-is?

Reply:
yeah, perfect